src/keydra/clients/splunk.py

```diff
@@ -280,17 +280,40 @@ def rotate_hectoken_cloud(self, inputname):
         config = json.loads(response['body'].read())['entry'][0]
         config.pop('token', None)
 
-        self._service.delete(
+        delresp = self._service.delete(
             '/services/dmc/config/inputs/__indexers/http/{}'.format(inputname),
             output_mode='json'
         )
-        createresp = self._service.post(
-            '/services/dmc/config/inputs/__indexers/http',
-            **config
-        )
-        newconfig = json.loads(createresp['body'].read())['entry'][0]
 
-        return newconfig['content']['token']
+        if delresp.status != 200:
```

Comment:
I see the other providers (bitbucket, cloudflare, github) just call `raise_for_status`, which seems simpler. Do we prefer custom logs?

Reply:
In this case yes. I'm wanting to get more detail from the response than just the error.